app/car.py

```diff
@@ -0,0 +1,21 @@
+import dataclasses
+import math
+from typing import List
+
+
+@dataclasses.dataclass
```

Comment:
It’s picky but it would be better to import `from dataclasses import dataclass`
And decorate `@dataclass`
Please implement all over the project